### ISSUE TITLE
Support flower breeding

### ISSUE THREAD

**Issue by lexicalunit** [theastropath/turbot#17]
Flowers follow real life genetics: https://docs.google.com/document/d/1ARIQCUc5YVEd01D7jtJT9EEJF45m07NXhAm4fOpNvCs/preview

**Comment by theastropath** [comment]
What kind of flower breeding functionality would be useful for the bot?  Calculating possible outcomes given two input sets of genes?

**Comment by lexicalunit** [comment]
Yeah not sure, maybe just a bot that has all the info and you can request specific info like "!flower red tulip" and it'd be like "Oh for red tulips you're gonna need blah blah..." 🤷‍♀️

Maybe Panda would have a better idea of what would be useful to track and get information on here.

**Comment by theastropath** [comment]
![image](https://user-images.githubusercontent.com/13684088/79622028-97be8b00-80e3-11ea-9dee-b3f534b1b12c.png)

**Comment by lexicalunit** [comment]
Another good source for flowers might be: https://animalcrossing.fandom.com/wiki/Flower

**Comment by lexicalunit** [comment]
**LOTS** of details here: https://docs.google.com/document/d/1XPAaPUvGMWQ-n5sBLV3jM78QKxqKjiGCzx5HnD7K5RI/preview

**Comment by lexicalunit** [comment]
Also this:
![BLUEROSEGUIDE](https://user-images.githubusercontent.com/1903876/80294651-a29aa080-871f-11ea-8022-6f4c4117546a.png)
